tool-openssl/c_rehash.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+# Get absolute path to script directory
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd)
```

Comment:
Unless there's reason not to, these bash scripts should start by ensuring that any failed commands also cause the script to fail:
```
set -ex
set -o pipefail
```

Reply:
Can combine them into one line as well `set -exo pipefail`